package_up

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+zip_filename=instacart-pp-$(whoami).zip
+pwd=$(pwd)
+
+
+echo "-- cleaning build files"
+rm -rf build
```

Comment:
The are build files in the module folder too, right? Maybe we should run `./gradlew clean` instead?

Reply:
i suppose this is why they say code reviews are a good thing :P. 🙏